### PR TITLE
Add Zenodo and SH badge for code archives

### LINKED ISSUE #179: Update citation with code archives
The current version of PBxplore has evolved from the original version published in 2017, notably by the dropout of Python < 3.6.

We should update the CITATION section in the README to specify the exact version that was used for the paper and link to code archives such as Zenodo and Software Heritage.

## Zenodo
Identifier
```
10.5281/zenodo.1016257
```
Badge [DOI](https://doi.org/10.5281/zenodo.1016257)
```
.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.1016257.svg
   :target: https://doi.org/10.5281/zenodo.1016257
```

## Software Heritage
Identifier
```
swh:1:dir:4260527877ab457f65eb1299437cf022301a4788;origin=https://github.com/pierrepo/PBxplore;visit=swh:1:snp:fb4066f408260c44decc7cac3624b56747a3de69;anchor=swh:1:rev:ba1290912cd65c86a01f70716e1d3133778bba75
```
Badge [SWH](https://archive.softwareheritage.org/swh:1:dir:4260527877ab457f65eb1299437cf022301a4788;origin=https://github.com/pierrepo/PBxplore;visit=swh:1:snp:fb4066f408260c44decc7cac3624b56747a3de69;anchor=swh:1:rev:ba1290912cd65c86a01f70716e1d3133778bba75/)
```
.. image:: https://archive.softwareheritage.org/badge/swh:1:dir:4260527877ab457f65eb1299437cf022301a4788/
    :target: https://archive.softwareheritage.org/swh:1:dir:4260527877ab457f65eb1299437cf022301a4788;origin=https://github.com/pierrepo/PBxplore;visit=swh:1:snp:fb4066f408260c44decc7cac3624b56747a3de69;anchor=swh:1:rev:ba1290912cd65c86a01f70716e1d3133778bba75/
```

I will handle this as soon as PR #177  will be merged and issue #178 fixed.

### PR DESCRIPTION
**Pull request by pierrepo**
Fix #179

### CODE REVIEW

**Comment by pierrepo** [comment]
@HubLot do you now how we could obtain something like this:

Zenodo image

in ReST (text and image on the same line) ?

**Comment by HubLot** [comment]
No, not really. Have you tried playing with the `:width:` keyword ? (like `:width: 30%`)

File: README.rst [hunk 1]
```diff
@@ -92,6 +92,20 @@ If you use PBxplore, please cite this tool as:
 | PBxplore: a tool to analyze local protein structure and deformability with Protein Blocks
 | *PeerJ*  5:e4013 `<https://doi.org/10.7717/peerj.4013>`_ (2017).
 
+The published version (1.3.8) is archived in
```
Comment (HubLot) [inline README.rst:95]: Add a new blank line above.
Currently, the "The plublished..." line is sticked to the reference one

Comment (pierrepo) [reply README.rst]: Thanks @HubLot 
Fixed!

**Comment by pierrepo** [comment]
`:width:` dit not help. The solution for inline image is to used [Substitution Definitions](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#substitution-definitions).

It should be OK now.

**Comment by HubLot** [comment]
great!

**Review by HubLot — APPROVED** [review]

**Pull request merged**